dashboard/app/controllers/learning_goal_evaluations_controller.rb

```diff
@@ -0,0 +1,35 @@
+class LearningGoalEvaluationsController < ApplicationController
+  def create
+    learning_goal_evaluation = LearningGoalEvaluation.new(learning_goal_evaluation_params)
+
+    learning_goal_evaluation.save!
+
+    render json: learning_goal_evaluation
+  end
+
+  def update
+    params.require(:id)
+    learning_goal_evaluation = LearningGoalEvaluation.find_by_id(params[:id])
+    if learning_goal_evaluation
+      learning_goal_evaluation.update(learning_goal_evaluation_params)
+      render json: learning_goal_evaluation
+    else
+      render status: :bad_request, json: learning_goal_evaluation
```

Comment:
nit: use `:not_found` instead of `bad_request` (`not_found` is a bit more descriptive)